modules/distributor/distributor.go

```diff
@@ -351,6 +362,12 @@ func (d *Distributor) PushBatches(ctx context.Context, batches []*v1.ResourceSpa
 		d.generatorForwarder.SendTraces(ctx, userID, keys, rebatchedTraces)
 	}
 
+	if len(d.cfg.Forwarders) > 0 && len(d.overrides.Forwarders(userID)) > 0 {
+		if err := d.forwardersManager.ForTenant(userID).ForwardBatches(ctx, tempopb.Trace{Batches: batches}); err != nil {
+			_ = level.Warn(d.logger).Log("msg", "failed to forward batches for tenant=%s: %w", userID, err)
```

Comment:
we should metric this in a way we can expose to grafana cloud tracing customers. `tempo_warnings_total` is already piped back to the user and was meant to be a place for generic warnings.

Reply:
Frankly, I don't understand. Do you mean that I cannot use warn-level logs because there is some random side-effect Could you provide some more context?

Reply:
If we don't add a metric to this PR, it might be handy to have an issue to make sure we come back to it.  A distributor metric might be handy for us internally too.